src/main/java/Duke.java

```diff
@@ -1,10 +1,352 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static duke.task.Task.Type.*;
+
 public class Duke {
```

Comment:
Your current Duke class seems to be doing alot of work. You should consider having separate classes for command processing, file IO, task management and so on. Keep your main class as clean as possible